.gitignore

```diff
@@ -14,7 +14,6 @@ dist/
 downloads/
 eggs/
 .eggs/
-lib/
```

Comment:
have you actually had an issue with this `lib/` directory being git-ignored on a case-insensitive filesystem?

I think it comes from this example .gitignore file for python projects from Github itself: 
https://github.com/github/gitignore/blob/main/Python.gitignore#L17

/cc @madig who authored the original change

Reply:
There is no "lib" folder, only a "Lib" folder. And we like to keep that. 
A new file inside that Lib folder will not show up in the git listing.

Reply:
ah ok yes, I see now, thanks
let's merge